src/main/resources/bundles/es/header.properties

```diff
@@ -92,7 +92,7 @@ dataIntegrity = Integridad de los datos
 manageProjects = Gestionar proyectos
 researchProjects = Proyectos de investigacion
 
-extendButton = Extender Sesi�n
+extendButton = Extender Sesi\\xf3n
```

Comment:
I can't sort out what this character is. Is this a typo?

Reply:
yeah it was typo i have fixed it extendButton = Extender Sesi\u00F3n